### PR TITLE
Swift: add 5.6.1 release

### PR DESCRIPTION
**Pull request by compnerd**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/57288)

### CODE REVIEW

File: manifests/s/Swift/Toolchain/5.6.1/Swift.Toolchain.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+PackageVersion: "5.6.1"
+PackageLocale: en-US
+Publisher: dt.compnerd.org
+# PublisherUrl: 
+# PublisherSupportUrl: 
+# PrivacyUrl: 
+Author: Swift Open Source Project
+PackageName: Swift
+PackageUrl: https://www.swift.org/
+License: Copyright (c) Swift Open Source Project
```
Comment (Trenly) [inline manifests/s/Swift/Toolchain/5.6.1/Swift.Toolchain.locale.en-US.yaml:14]: A copyright is not a license; Can you please update this to the actual license terms such as `Proprietary`, `MIT`, `GPL-2.0`, or whatever license applies

File: manifests/s/Swift/Toolchain/5.6.1/Swift.Toolchain.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+# LicenseUrl: 
+# Copyright: 
+# CopyrightUrl: 
+ShortDescription: Swift Toolchain for Windows
+# Description: 
+Moniker: swift
+Tags:
+- swift
```
Comment (Trenly) [inline manifests/s/Swift/Toolchain/5.6.1/Swift.Toolchain.locale.en-US.yaml:22]: This tag is redundant with the moniker and may not do anything

**Review by Trenly — CHANGES_REQUESTED** [review]

File: manifests/s/Swift/Toolchain/5.6.1/Swift.Toolchain.locale.en-US.yaml (hunk 1 above)
Comment (compnerd) [reply manifests/s/Swift/Toolchain/5.6.1/Swift.Toolchain.locale.en-US.yaml:14]: I'm not sure how this has managed to stay there for so long.  Thank you for noticing the issue!

File: manifests/s/Swift/Toolchain/5.6.1/Swift.Toolchain.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,27 @@
+# Created with YamlCreate.ps1 v2.1.0 $debug=AUSU.5-1-22567-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Swift.Toolchain
+PackageVersion: "5.6.1"
+PackageLocale: en-US
+Publisher: dt.compnerd.org
```
Comment (ItzLevvie) [inline manifests/s/Swift/Toolchain/5.6.1/Swift.Toolchain.locale.en-US.yaml:7]:
```suggestion
Publisher: swift.org
```
to match Control Panel.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**